tools/doc-shredder/processors/shredMapProcessor.js

```diff
@@ -82,13 +94,24 @@ module.exports = function shredMapProcessor(log, createDocMessage) {
   }
 };
 
-function getExampleName(fragPath) {
-  // pattern to isolate base fileName and extension from fragment name
-  var rx = /(.*)\-(.*)\.(.s)/;
-  var r = rx.exec(fragPath);
-  if (r) {
-    return r[1] + '.' + r[3];
-  } else {
-    return fragPath;
-  }
+// TODO: use the functionality in public/resources/js/util.js once it lands.
+function jadeBaseFileNameToExampleName(name) {
+  // Adjust for known cases where chapter name is not the example name.
```

Comment:
Kind of a PITA I know.  I wanted to fix the folders so we didn't have the discrepancy that requires this hack. That would mean adding firebase rewrite rules to handle external links. Sigh. I guess this gets it done. But it adds technical debt.